ArduPlane/Plane.h

```diff
@@ -1105,6 +1105,14 @@ class Plane : public AP_Vehicle {
     bool ekf_over_threshold();
     void failsafe_ekf_event();
     void failsafe_ekf_off_event(void);
+    
+    enum Crow_Mode {
```

Comment:
I got a little confused with using it for, yaw, maybe this could be Crow_Flap_Mode

Reply:
Yaw is independently controlled with DSPOILR_RUD_RATE....on a non flying wing you would set it to "0"....the wiki was using CROW for these actions....so I kept it consistent....I have no bias one way or the other...

Reply:
Maybe its just me getting confused between Dspoilers and Crow flaps, feel free to do whatever

Reply:
enum class Crow_mode

Reply:
changed to enum class